[2024-03-18T08:03:35.326+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sftp_sync.sftp_sensor_task manual__2024-03-18T08:03:31.649964+00:00 [queued]>
[2024-03-18T08:03:35.343+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sftp_sync.sftp_sensor_task manual__2024-03-18T08:03:31.649964+00:00 [queued]>
[2024-03-18T08:03:35.344+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-18T08:03:35.362+0000] {taskinstance.py:2217} INFO - Executing <Task(SFTPSensor): sftp_sensor_task> on 2024-03-18 08:03:31.649964+00:00
[2024-03-18T08:03:35.391+0000] {standard_task_runner.py:60} INFO - Started process 1344 to run task
[2024-03-18T08:03:35.395+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sftp_sync', 'sftp_sensor_task', 'manual__2024-03-18T08:03:31.649964+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/sftp_sync_between_source_and_target.py', '--cfg-path', '/tmp/tmp8n0ik_t7']
[2024-03-18T08:03:35.398+0000] {standard_task_runner.py:88} INFO - Job 79: Subtask sftp_sensor_task
[2024-03-18T08:03:35.476+0000] {task_command.py:423} INFO - Running <TaskInstance: sftp_sync.sftp_sensor_task manual__2024-03-18T08:03:31.649964+00:00 [running]> on host fa05d1b87173
[2024-03-18T08:03:35.604+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sftp_sync' AIRFLOW_CTX_TASK_ID='sftp_sensor_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T08:03:31.649964+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T08:03:31.649964+00:00'
[2024-03-18T08:03:35.615+0000] {base.py:83} INFO - Using connection ID 'source_sftp' for task execution.
[2024-03-18T08:03:35.616+0000] {sftp.py:80} INFO - Poking for /home/***/filesystem/, with pattern *
[2024-03-18T08:03:35.618+0000] {ssh.py:301} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-03-18T08:03:35.804+0000] {transport.py:1909} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-03-18T08:03:36.041+0000] {transport.py:1909} INFO - Authentication (password) successful!
[2024-03-18T08:03:36.160+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-03-18T08:03:36.176+0000] {sftp.py:97} INFO - Found File /home/***/filesystem/file1.txt last modified: 20240318053950
[2024-03-18T08:03:36.178+0000] {sftp.py:169} INFO - [chan 0] sftp session closed.
[2024-03-18T08:03:36.179+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2024-03-18T08:03:36.249+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=sftp_sync, task_id=sftp_sensor_task, execution_date=20240318T080331, start_date=20240318T080335, end_date=20240318T080336
[2024-03-18T08:03:36.345+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T08:03:36.506+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
